README.md

```diff
@@ -85,9 +85,13 @@ Generate Release Notes action is dedicated to enhance the quality and organizati
 - **Default**: false
 
 ### `skip-release-notes-label`
```

Comment:
Name changed to: skip-release-notes-label**S**

Reply:
Fixed in commit b4eacda582e8811772da2a9f4633bb3de46c9ce6.